lib/menu.rb

```diff
@@ -15,15 +15,13 @@ def show
   end
 
   def check(dish)
-    _dish = dish.to_sym
+    the_dish = dish.to_sym
     @dishes.each do |dish|
-      "food #{dish[:food]}"
-      if _dish == dish[:food]
+      if the_dish == dish[:food]
```

Comment:
I feel this could be simplified to a ternary operator

ie 'the_dish == dish[:food] ? true : false